src/Nexcess/GithubWebhookJira/Webhook.php

```diff
@@ -144,11 +174,15 @@ private function _getData() : \stdClass {
    * @return void
    */
   public function process() {
+    if (! isset($this->_getData()->pull_request)) {
+      return;
+    }
+
     switch ($this->_getData()->action) {
-      case self::ACTION_REOPENED:
-        /* falls through */
       case self::ACTION_EDITED:
```

Comment:
For the edit case need to:
1. update the PR message as needed
2. transition jira issues based on the PR state